src/index.html

```diff
@@ -7,12 +7,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac27 Retina 5K Monoblock (MNED2UA/A)"
+      />
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="card__productCode">Product code: 195434</p>
+      <div class="card__starsAndReviews">
+        <ul class="stars">
```

Comment:
read the checklist again carefully
![image](https://github.com/mate-academy/layout_product-cards/assets/106397016/03235bc5-d71c-41f0-9a5a-33fff457ee3c)

Reply:
> read the checklist again carefully ![image](https://private-user-images.githubusercontent.com/106397016/341796145-03235bc5-d71c-41f0-9a5a-33fff457ee3c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5ODg2NjYsIm5iZiI6MTcxODk4ODM2NiwicGF0aCI6Ii8xMDYzOTcwMTYvMzQxNzk2MTQ1LTAzMjM1YmM1LWQ3MWMtNDFmMC05YTVhLTMzZmZmNDU3ZWUzYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyMVQxNjQ2MDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZDMzNmQyYjFmZDE3YjY0YTkzYzAxMDk3OTZiYjFlNWMxMTU5ZjJiNWFiYjlkZTJjODdiZmM1YTU5YmVmNWYzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.m0hntq8aCiyz1MUrEoDXQJOITGx5hTf9LWWUVaiyB5I)

Not fixed, you should show the corresponding amount of active stars depending on the modifier that you set to the stars block